library/Fission/CLI/Command/Login.hs

```diff
@@ -70,9 +68,13 @@ login = do
 
       case authResult of
         Right peers -> do
-          let writeTo = UserConfig {username = username
-                                    , password = (BS.pack password)
-                                    , peers = peers}
+          let writeTo = UserConfig { username = username
+                                   , password = BS.pack password
+                                   , peers    = NonEmpty.fromList peers
```

Comment:
`NonEmpty` is a data type like `[]`, but always has a value in it (i.e. `head` is safe to use without a `Maybe`). My putting this upstream, we can always assume that the value is there

Reply:
Really we should be using `nonEmpty` here, since it's total 🤔